stravalib/protocol.py

```diff
@@ -163,7 +163,7 @@ def refresh_access_token(self, client_id, client_secret, refresh_token):
 
         return access_info
 
-    def _resolve_url(self, url):
+    def resolve_url(self, url):
```

Comment:
is this a helper? i'm curious is the rename just for clarity? i'm fine either way. i'm used to [helper  / non public methods using an underscore to follow PEP8 ](https://peps.python.org/pep-0008/#method-names-and-instance-variables)

Reply:
I decided to reuse this method outside of the `ApiV3` class to help set up the mocks, so made this method public. Alternatively, we could also refactor by completely moving the server address, API base, and resolve_url function to some generic util module or package __init__ module. Happy to make that change if you think the latter is necessary.

Reply:
@jsamoocha got it. this makes sense. I will default to your expertise here in setting this up. What would be the benefit of a complete refactor?

thank you again for this work here. it's awesome to have you here helping maintain stravalib.